vector/src/main/res/layout/item_room.xml

```diff
@@ -112,13 +112,27 @@
         app:layout_constraintTop_toBottomOf="@id/roomAvatarContainer"
         tools:layout_marginStart="20dp" />
 
+    <Space
+        android:id="@+id/topMarginSpace"
+        android:layout_width="0dp"
+        android:layout_height="12dp"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <Space
+        android:id="@+id/centerTitleSpace"
+        android:layout_width="match_parent"
+        android:layout_height="15dp"
+        android:visibility="gone"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/topMarginSpace" />
```

Comment:
weird way to align text vertically. If the font size is changed, it will not work properly.
Is is possible to do better using ConstraintLayout attributes?
Maybe adding a new sublayout if there is no easier solution.

Reply:
I tried this with changing consraints programatically, but that affects other bindings as you scroll. There's no easy way to 'reset' the view to its original xml state.

Can you explain more what you mean by sublayout?

Reply:
sublayout: add a new ConstraintLayout to pack the Title and the Subtitle.
I did not check if this is possible, there are other views around.
I will try to play a bit on this layout to see if I find something better.

Reply:
I think the best way to do this imo is to inflate an entirely different Epoxy item based on whether there is a subtitle or not. What are your thoughts on that?

Reply:
I will improve that in the future, there is another bug related, with the placeholder.

Reply:
https://github.com/vector-im/element-android/pull/5860/commits/03acf4505acba15f48ae9e8783829aa4087e0c61

Reply:
Thanks. I am removing myself from the reviewer now.